presto-spi/src/main/java/com/facebook/presto/spi/ConnectorSession.java

```diff
@@ -35,5 +35,8 @@ default String getUser()
 
     long getStartTime();
 
+    @Deprecated
+    boolean isLegacyTimestamp();
```

Comment:
@martint Are you ok with this being in ConnectorSession. I feel like it's necessary and we don't have a choice.